@codex please fix

```
 building client (vite) 
 ✓ tests/navigateWithRetry.test.ts (2 tests) 19ms
 ✓ tests/k8sEnvironmentOverlays.test.ts (2 tests) 5ms
 ✓ tests/noConflictMarkers.test.ts (1 test) 1253ms
   ✓ repository sanity ‑ no merge-conflict markers > contains no <<<<<<< or >>>>>>> markers  1250ms
 ✓ scripts/tests/questDependencies.test.ts (2 tests) 7ms
 ✓ tests/svelteResolution.test.ts (4 tests) 503ms
 ✓ tests/architectureNotesAccessibilityBaseline.test.ts (2 tests) 4ms
 ✓ tests/questDependencyReferences.test.ts (1 test) 42ms
 ✓ tests/questIdPathConsistency.test.ts (1 test) 29ms
 ✓ tests/svelteSubpathResolver.test.ts (2 tests) 10ms
 ✓ tests/serviceWorkerConfigCache.test.ts (1 test) 5ms
 ✓ tests/noDuplicateItemIds.test.ts (1 test) 5ms
 ✓ frontend/tests/docsMarkdownSafety.test.ts (1 test) 9ms
 ✓ tests/questReliabilityCoverage.test.ts (1 test) 191ms
 ✓ tests/v3ReleaseStateDoc.test.ts (3 tests) 7ms
 ✓ frontend/tests/questListClassifier.test.ts (3 tests) 6ms
 ✓ tests/changelogQuestBranchingNote.test.ts (1 test) 7ms
 ✓ tests/metricsEndpoint.test.ts (3 tests) 23ms
 ✓ tests/docsSearchFeatures.test.ts (2 tests) 6ms
 ✓ tests/questProcessRecoveryPaths.test.ts (1 test) 141ms
 ✓ tests/questsPolishPromptSections.test.ts (3 tests) 6ms
 ✓ tests/builtinQuestSchema.test.ts (1 test) 89ms
 ✓ tests/githubTokenPersistence.test.ts (1 test) 11ms
 ✓ tests/spellcheckConfig.test.ts (1 test) 10ms
 ✓ frontend/src/components/__tests__/QuestLinkChips.spec.ts (2 tests) 172ms
06:38:05 [build] Rearranging server assets...
06:38:05 [build] Server built in 35.78s
06:38:05 [build] Complete!
 ✓ tests/buildOutputValidation.test.ts (1 test) 42649ms
   ✓ build output validation > should not contain /src/scripts/ paths in built files  42647ms
 ✓ tests/pnpmFrontendFilter.test.ts (1 test) 19ms
 ✓ tests/scanSecretsScript.test.ts (2 tests) 77ms
 ✓ tests/helmChartReleaseVersion.test.ts (1 test) 4ms
 ✓ tests/changelogDocsAICoPilots.test.ts (1 test) 5ms
 ✓ tests/compactItemListHelpers.test.ts (2 tests) 5ms
 ✓ tests/userJourneysCoverage.test.ts (1 test) 6ms
 ✓ tests/customContentLoadTest.test.ts (2 tests) 1343ms
   ✓ runCustomContentLoadTest > uses default count when omitted  1310ms
 ✓ tests/measureEcSolutionRegression.test.ts (1 test) 7ms
 ✓ tests/changelogDocsContentUpdate.test.ts (1 test) 7ms
 ✓ scripts/tests/detectZeroTests.test.ts (5 tests) 6ms
 ✓ tests/customQuestLoader.test.ts (1 test) 19ms
 ✓ tests/raspiSecretsDocs.test.ts (2 tests) 6ms
 ✓ scripts/tests/preparePrScripts.test.ts (1 test) 5ms
 ✓ tests/chartResourcesDefaults.test.ts (1 test) 27ms
 ✓ tests/inject-ids.test.ts (1 test) 6ms
 ✓ tests/helmChartGracefulShutdown.test.ts (2 tests) 4ms
 ✓ tests/cheatsAvailability.test.ts (5 tests) 9ms
 ✓ tests/changelogCloudSync.test.ts (1 test) 5ms
 ✓ tests/docsRagDrift.test.ts (2 tests) 474ms
   ✓ docs RAG drift intent > forces the v3 release state doc for v2-only drift queries  458ms
 ✓ tests/changelogProcessRequirementsNote.test.ts (1 test) 6ms
 ✓ tests/markdownEncodingFix.test.ts (3 tests) 5ms
 ✓ tests/helmChartPersistence.test.ts (2 tests) 5ms
 ✓ tests/quest-title-unique.test.ts (3 tests) 4ms
 ✓ frontend/src/components/__tests__/RemainingTime.spec.ts (2 tests) 50ms
 ✓ tests/questCopyRegression.test.ts (1 test) 5ms
 ✓ scripts/tests/validateItem.test.ts (2 tests) 8ms
 ✓ tests/chatPersonas.test.ts (1 test) 143ms
 ✓ tests/workflowPnpmCache.test.ts (3 tests) 6ms
 ✓ tests/changelogNpcChatNote.test.ts (1 test) 4ms
 ✓ frontend/tests/chatHints.test.ts (4 tests) 7ms
 ✓ tests/changelogProcessReservation.test.ts (1 test) 6ms
 ✓ tests/questTreesDoc.test.ts (3 tests) 5ms
 ✓ tests/deployEnvFluxDocs.test.ts (1 test) 5ms
 ✓ tests/networkPolicyObservability.test.ts (2 tests) 5ms
 ✓ tests/roadmapDocLinks.test.ts (1 test) 5ms
 ✓ tests/backupScript.test.ts (2 tests) 121ms
 ✓ tests/guildsDocs.test.ts (2 tests) 6ms
 ✓ scripts/tests/validateQuest.test.ts (2 tests) 29ms
 ✓ tests/seedCustomContent.test.ts (1 test) 30ms
 ✓ tests/menuToolbox.test.ts (2 tests) 6ms
 ✓ tests/roadmapPastYearsComplete.test.ts (1 test) 5ms
 ✓ tests/fix-markdown-image-paths.test.ts (1 test) 8ms
 ✓ tests/promptsCodexLinks.test.ts (1 test) 5ms
 ✓ tests/itemSelectorLocators.test.ts (1 test) 4ms
 ✓ tests/customContentItemsStore.test.ts (1 test) 17ms
 ✓ tests/helmChartProbes.test.ts (1 test) 5ms
 ✓ tests/userJourneysDoc.test.ts (2 tests) 4ms
 ✓ tests/buildWorkflowMultiArch.test.ts (2 tests) 5ms
 ✓ tests/processHardening.test.ts (1 test) 9ms
 ✓ tests/cacheVersionSync.test.ts (1 test) 6ms
 ✓ tests/dbBenchmarkCli.test.ts (1 test) 276ms
 ✓ tests/questSchemaValidation.test.ts (2 tests) 5ms
 ✓ tests/canvasDependencyAlignment.test.ts (1 test) 6ms
 ✓ tests/changelogDocsReleaseChecklist.test.ts (1 test) 4ms
 ✓ tests/playwrightConfigBrowsers.test.ts (1 test) 6ms
 ✓ frontend/src/components/__tests__/SearchBar.spec.ts (1 test) 53ms
 ✓ tests/customContentBackupImporterAccessibility.test.ts (1 test) 4ms
 ✓ tests/license.test.ts (3 tests) 7ms
 ✓ tests/questGuidelinesEditorDoc.test.ts (1 test) 5ms
 ✓ tests/ubiQuestLeaderboardCopy.test.ts (1 test) 4ms
 ✓ frontend/tests/playwrightConfigHost.test.ts (1 test) 6ms
 ✓ tests/questGraphValidation.test.ts (2 tests) 207ms
 ✓ tests/roadmap10xQuests.test.ts (1 test) 4ms
 ✓ tests/rocketsDocsContent.test.ts (1 test) 6ms
 ✓ tests/solarBasicsIcons.test.ts (5 tests) 9ms
 ✓ scripts/tests/lintStagedItemValidation.test.ts (1 test) 5ms
 ✓ scripts/tests/lintStagedQuestValidation.test.ts (1 test) 5ms
 ✓ tests/rocketsDocsPromise.test.ts (1 test) 3ms
 ✓ tests/fsChecksInvalidEncoding.test.ts (1 test) 7ms
 ✓ tests/roadmapSuborbitalLaunch.test.ts (1 test) 8ms
 ✓ tests/roadmapCustomQuests.test.ts (1 test) 6ms
 ✓ tests/changelogNotes.test.ts (1 test) 3ms
 ✓ tests/testingDocsPromises.test.ts (1 test) 5ms
 ✓ frontend/src/components/__tests__/compactItemListHelpers.spec.ts (1 test) 4ms
 ✓ tests/k3sChecklistComplete.test.ts (1 test) 3ms
 ✓ tests/docsFAQ.test.ts (1 test) 4ms
 ✓ tests/roadmapPromises.test.ts (1 test) 5ms
 ✓ tests/solarDocWind.test.ts (1 test) 6ms
 ✓ tests/docsPromptsOutages.test.ts (2 tests) 4ms
 ✓ tests/menu-active.test.ts (1 test) 3ms
 ✓ tests/sanitization.test.ts (1 test) 22ms
 ✓ tests/metricsFallback.test.ts (1 test) 50ms
 ✓ tests/itemSelectorTouch.test.ts (2 tests) 7ms
 ✓ tests/process-form-mobile.test.ts (1 test) 4ms
 ✓ tests/questCountIncrease.test.ts (1 test) 19ms
 ✓ tests/item-form-mobile.test.ts (1 test) 6ms
 ✓ tests/quest-form-mobile.test.ts (1 test) 5ms
 ✓ tests/titlesDoc.test.ts (1 test) 4ms
 ✓ tests/menu-leaderboard.test.ts (1 test) 6ms
 ✓ tests/togglePreviewId.test.ts (3 tests) 5ms
 ✓ scripts/tests/questQuality.test.ts (1 test) 5ms
 ✓ tests/npmTestCiScript.test.ts (1 test) 5ms
 ✓ tests/menu-settings.test.ts (1 test) 6ms
 ✓ tests/achievementsCatalog.test.ts (1 test) 6ms
 ✓ tests/mergePlanChecklist.test.ts (1 test) 5ms
 ✓ frontend/__tests__/ShoppingForm.test.js (1 test) 5ms
 ✓ tests/menu-stats.test.ts (1 test) 5ms
 ✓ tests/profileTitles.test.ts (1 test) 4ms
 ✓ frontend/tests/menu.test.ts (1 test) 3ms
 ✓ frontend/tests/duration.test.ts (2 tests) 4ms
 ✓ tests/menuTitles.test.ts (1 test) 6ms
 ✓ tests/pnpmfile.test.ts (1 test) 5ms
 ✓ tests/domExceptionGlobal.test.ts (1 test) 5ms
 ✓ tests/domexception-global.test.ts (1 test) 5ms
 ✓ tests/domexception-globals.test.ts (1 test) 5ms
 ✓ tests/sysadminQuest.test.ts (1 test) 6ms
 ✓ tests/shopIndex.test.ts (1 test) 6ms
 ✓ tests/eslintPluginPresence.test.ts (1 test) 4ms
 ✓ frontend/tests/gameStateFlush.test.ts (1 test) 5ms
 ✓ tests/checkoutFetchDepth.test.ts (1 test) 4ms
 ❯ scripts/tests/prettierFormatting.test.ts (1 test | 1 failed) 27926ms
   × Prettier formatting > frontend sources are formatted 27924ms
     → npm run format:check failed.

> dspace@3.0.0 format:check
> prettier --config .prettierrc --check "**/*.{js,ts,svelte,json,md}"

Checking formatting...
[warn] scripts/utils/ensure-playwright-browsers.js
[warn] Code style issues found in the above file. Run Prettier with --write to fix.

 ✓ tests/npmTestRunsAllSuites.test.ts (1 test) 5ms
 ✓ tests/astroConfigLogLevel.test.ts (1 test) 5ms
 ✓ tests/prismSvelte.test.ts (1 test) 13ms
 ✓ scripts/tests/dbBenchmarkScript.test.ts (1 test) 4ms
 ✓ tests/progressionBalance.test.ts (1 test) 28ms
 ✓ tests/domException.test.ts (1 test) 6ms
 ✓ tests/indexeddb.server.test.ts (1 test) 243ms
 ✓ frontend/tests/sanity.test.ts (1 test) 3ms

⎯⎯⎯⎯⎯⎯⎯ Failed Tests 1 ⎯⎯⎯⎯⎯⎯⎯

 FAIL  scripts/tests/prettierFormatting.test.ts > Prettier formatting > frontend sources are formatted
Error: npm run format:check failed.

> dspace@3.0.0 format:check
> prettier --config .prettierrc --check "**/*.{js,ts,svelte,json,md}"

Checking formatting...
[warn] scripts/utils/ensure-playwright-browsers.js
[warn] Code style issues found in the above file. Run Prettier with --write to fix.

 ❯ scripts/tests/prettierFormatting.test.ts:17:13
     15|       const stdout = error?.stdout?.toString?.() ?? '';
     16|       const stderr = error?.stderr?.toString?.() ?? '';
     17|       throw new Error(`npm run format:check failed.\n${stdout}${stderr…
       |             ^
     18|     }
     19|   }, 240000);

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[1/1]⎯


 Test Files  1 failed | 293 passed (294)
      Tests  1 failed | 1053 passed (1054)
   Start at  06:36:16
   Duration  163.70s (transform 9.77s, setup 11.30s, collect 46.92s, tests 118.90s, environment 193.23s, prepare 36.20s)


Error: Error: npm run format:check failed.

> dspace@3.0.0 format:check
> prettier --config .prettierrc --check "**/*.{js,ts,svelte,json,md}"

Checking formatting...
[warn] scripts/utils/ensure-playwright-browsers.js
[warn] Code style issues found in the above file. Run Prettier with --write to fix.

 ❯ scripts/tests/prettierFormatting.test.ts:17:13


 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1.
```

new codex task, not a r/e/v/i/e/w task

--- a/frontend/scripts/utils/ensure-playwright-browsers.js
+++ b/frontend/scripts/utils/ensure-playwright-browsers.js
@@ -207,7 +207,11 @@ export async function ensurePlaywrightBrowsers(options = {}) {
                 console.warn(
                     'PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1 but Playwright chromium browser is missing. E2E tests may fail.'
                 );
-            } else if (requireHeadlessShell && !hasHeadlessShell && !warnedMissingHeadlessShellPaths.has(executablePath)) {
+            } else if (
+                requireHeadlessShell &&
+                !hasHeadlessShell &&
+                !warnedMissingHeadlessShellPaths.has(executablePath)
+            ) {
                 warnedMissingHeadlessShellPaths.add(executablePath);
                 console.warn(
                     `PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1 but Playwright chromium headless shell is missing for ${executablePath}. E2E tests may fail.`
@@ -247,7 +251,10 @@ export async function ensurePlaywrightBrowsers(options = {}) {
     });
 
     const postInstallAssets = getChromiumAssetStatus(browser);
-    if (!postInstallAssets.hasChromium || (requireHeadlessShell && !postInstallAssets.hasHeadlessShell)) {
+    if (
+        !postInstallAssets.hasChromium ||
+        (requireHeadlessShell && !postInstallAssets.hasHeadlessShell)
+    ) {
         const missingAsset = !postInstallAssets.hasChromium
             ? 'chromium executable'
             : 'chromium headless shell';

